website/package.json

```diff
@@ -6,43 +6,40 @@
     "version": "3.3.7"
   },
   "dependencies": {
-    "autoprefixer": "6.4.0",
-    "cssnano": "3.7.3",
-    "hexo": "^3.3.7",
-    "hexo-browsersync": "0.2.0",
-    "hexo-deployer-git": "0.2.0",
+    "autoprefixer": "^7.2.5",
+    "cssnano": "^3.10.0",
+    "hexo": "^3.4.4",
+    "hexo-browsersync": "^0.3.0",
+    "hexo-deployer-git": "^0.3.1",
     "hexo-generator-alias": "^0.1.3",
-    "hexo-generator-archive": "0.1.4",
-    "hexo-generator-category": "0.1.3",
-    "hexo-generator-feed": "1.2.0",
-    "hexo-generator-index": "0.2.0",
-    "hexo-generator-tag": "0.2.0",
-    "hexo-renderer-ejs": "0.2.0",
-    "hexo-renderer-marked": "0.3.0",
-    "hexo-renderer-postcss": "https://github.com/arturi/hexo-renderer-postcss#afca2bc12f5816067b15a9d24017c70e077b9f0b",
-    "hexo-renderer-scss": "1.0.2",
+    "hexo-generator-archive": "^0.1.5",
+    "hexo-generator-category": "^0.1.3",
+    "hexo-generator-feed": "^1.2.2",
+    "hexo-generator-index": "^0.2.1",
+    "hexo-generator-tag": "^0.2.0",
+    "hexo-renderer-ejs": "^0.2.0",
+    "hexo-renderer-marked": "^0.3.2",
+    "hexo-renderer-postcss": "https://github.com/goto-bus-stop/hexo-renderer-postcss#new-postcss",
```

Comment:
Change this back after https://github.com/arturi/hexo-renderer-postcss/pull/4 is merged